config/vufind/ProQuestFSG.ini

```diff
@@ -0,0 +1,160 @@
+; costs a lot of performance and most users are not very likely to navigate
+; further down than 20 pages of a search result.
+; This is especially useful to prevent webcrawlers from descending too deep and
+; eating up search backend performance. 
+; Default is 400; the API supports a maximum limit of 1000.
+;result_limit = 400
+
+; These are the default recommendations modules to use when no specific setting
+; are found in the [TopRecommendations], [SideRecommendations] or
+; [NoResultsRecommendations] sections below.
+; See the comments above those sections for details on legal settings.  You may
+; repeat these lines to load multiple recommendations.
+;default_top_recommend[] = TopFacets:FacetsTop:Primo
+default_side_recommend[] = SideFacets:Facets:CheckboxFacets:ProQuestFSG
+;default_noresults_recommend[] = SwitchTab
+default_noresults_recommend[] = RemoveFilters
+
+; When you filter a search using facets, should VuFind retain your current filters
+; on the next search and provide a reset button to clear them (true), or should it
+; always perform new searches unfiltered (false)?
+retain_filters_by_default = true
+
+; Whether to always (when there are active filters) display the "Reset Filters"
+; button regardless of the retain_filters_by_default setting. Default is false.
+always_display_reset_filters = false
+
+; This controls whether results are loaded with JavaScript when paging or changing
+; settings. Loading results this way improves performance and accessibility, and is
+; enabled by default. Loading will automatically fall back to non-JS mode if
+; JavaScript is not supported by the browser.
+load_results_with_js = true
+
+; This setting can be used to configure pagination control on top of results.
+; Possible options are:
+; empty string or false  No top pagination (default)
+; simple                 Simple next/prev button pagination
+; full                   Full pagination alike to the one at the bottom of results
+;top_paginator = simple
+
+; The following two sections can be used to associate specific recommendations
+; modules with specific search types defined in the [Basic_Searches] section
+; below.  For all the details on how these sections work, see the comments above
+; the equivalent sections of searches.ini.  Recommendations work the same in
+; ProQuestFSG as they do in the regular Search module.
+;[SideRecommendations]
+; No search-specific settings by default -- add your own here.
+;[TopRecommendations]
+; No search-specific settings by default -- add your own here.
+
+; The order of display is as shown below
+; The name of the index field is on the left
+; The display name of the field is on the right
+[Facets]
+Databases  = "Source"
+
+; Checkbox facets are facets, which are getting displayed as checkboxes
+; pcAvailability is a special facet. It's used to show all records found in
+; PrimoCentral without checking the local availability against a holdingsfile.
```

Comment:
This comment also seems to be copied from elsewhere and probably not relevant here.

Reply:
Thanks, yes clearly using Primo.ini as a source was not ideal.  I've reverted this to (almost) what it says in facets.ini.